jobs/config.json

```diff
@@ -9992,7 +9992,7 @@
       "--cluster=",
       "--deployment=kubernetes-anywhere",
       "--env-file=jobs/platform/gce.env",
-      "--extract=ci/latest-1.8",
+      "--extract=latest-1.8",
```

Comment:
this is kubernetes tarball location though, are you meant to change kubeadm version?

Reply:
I change the kubeadm version being upgraded to in the --upgrade_args. With this line, I am hoping also run tests at a matching version (ie. latest-1.8)

Reply:
I think --extract=latest-1.8 is not supported in https://github.com/kubernetes/test-infra/blob/master/kubetest/extract.go, but correct me if I'm wrong

Reply:
release/latest-1.8 maybe?